pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
```

Comment:
Does this apply just to local tags or remote also? Maybe it would be nice to mention that.